runners/direct-java/pom.xml

```diff
@@ -125,6 +125,23 @@
             </execution>
           </executions>
         </plugin>
+        <!-- builds unshaded JARs used by Cloud Dataflow -->
```

Comment:
?

Is this used by internal tests, or something? the Dataflow runtime shouldn't really be depending on the `DirectRunner`

Reply:
I believe that it's used by internal tests, yes.